testdata/generate.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
Probably it is worth to integrate script with Makefile or even move command to it. However it's up to you.

Reply:
I've added a ```testdata``` target into Makefile
```patch
+.PHONY: testdata
+testdata:
+       (cd ./testdata; ./generate.sh)
+
```

Reply:
Seems resolved, so I'll resolve the discussion.